slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -214,6 +216,13 @@ class SLAMServiceImpl final : public SLAMService::Service {
     // or not the pointcloud string contains any points.
     bool ExtractPointCloudToString(std::string &pointcloud);
 
+    // GetLatestPointCloudMapString paints and returns the latest map as a pcd
+    // string with probability estimates written to the color channel
+    bool GetLatestPointCloudMapString(std::string &pointcloud);
+
+    // For a given color channel convert the scale from 102-255 to 100-0
+    int ViamColorToProbability(int color);
```

Comment:
even though using Viam in the name is redundant I still wanted to include it because this is our specific calibration. Fine to change the name if people believe that is unecessary